modules/RoyalCommands/src/main/resources/config.yml

```diff
@@ -689,3 +689,25 @@ items:
       # The lore to apply to spawned items. This allows for easy identification without /spawninfo.
       lore:
       - "&9Spawned"
+
+###
+# Death Message Options #
+###
+death:
+  # "use_custom_death" - Should custom death messages be used?
+  use_custom_death: true
+  # "show_interworld" - Should death messages from other worlds be displayed to players?
+  show_interworld: true
+  # "disabled_worlds" - List of worlds to disable death messages in
+  disabled_worlds:
+    - someworldhere
+  ###################
+  # Colors!
+  # The default colors are HeroicDeath colors (if I recall correctly)
+  # You can change them here
+  # "var_color" is the color of the variables (those listed above)
+  # "mes_color" is the color of the rest of the message.
+  # Get all the colors here: http://jd.bukkit.org/apidocs/org/bukkit/ChatColor.html
+  ###################
+  var_color: DARK_AQUA
+  mes_color: RED
```

Comment:
I checked, WMA uses

```
var_color: RED
mes_color: GRAY
```

I think they're more suitable colours.

Originally, I was going to suggest moving these two into a `death.colors` subcategory, but actually I'm thinking let's use `messages.options.negative` and `messages.options.neutral` directly and scrap death-specific colours. Thoughts?

Reply:
I like the idea of this, I will switch them to use `messages.options.colors.negative` and `messages.options.colors.positive`

Reply:
This was done in baf1802